functions.php

```diff
@@ -218,37 +218,37 @@ function odm_jeo_scripts()
   endif;
 
   if (is_page( array( 'map-explorer', 'embed' )) || is_singular('map') || is_singular('map-layer') || is_singular('profiles') || is_home()):
-    
+
     if ( file_exists( STYLESHEETPATH . '/inc/jeo-scripts/jeo.js')):
        wp_deregister_script('jeo');
        wp_enqueue_script('jeo', get_stylesheet_directory_uri() . '/inc/jeo-scripts/jeo.js', array('mapbox-js', 'underscore', 'jquery'), '0.5.0');
     endif;
 
-    if ( file_exists( STYLESHEETPATH . '/inc/jeo-scripts/fullscreen.js')):
-       wp_deregister_script('jeo.fullscreen');       
+    if ( file_exists( STYLESHEETPATH . '/inc/jeo-scripts/control_fullscreen.js')):
+       wp_deregister_script('jeo.fullscreen');
     endif;
-    
-    if ( file_exists( STYLESHEETPATH . '/inc/jeo-scripts/clearscreen.js')):
-       wp_deregister_script('jeo.clearscreen');       
+
+    if ( file_exists( STYLESHEETPATH . '/inc/jeo-scripts/control_clearscreen.js')):
+       wp_deregister_script('jeo.clearscreen');
     endif;
-    
+
     wp_enqueue_script('jeo.fullscreen', get_stylesheet_directory_uri() . '/inc/jeo-scripts/control_fullscreen.js',array('jeo'), '0.2.0');
     wp_enqueue_script('jeo.clearscreen', get_stylesheet_directory_uri() . '/inc/jeo-scripts/control_clearscreen.js', array('jeo'), '1.0.0');
```

Comment:
Also, the **control_clearscreen.js** should only be made accessible while on desktop. On mobile the **map layers** box is already hidden and can only be shown by clicking the layers button